### DISCUSSION TITLE
Invalid copilot token: missing token: 403

### DISCUSSION THREAD

**Discussion by jose1500s** [community/community#51807] [Copilot Conversations]
### Select Topic Area

Question

### Body

A few days ago I was using copilot without problems, 3 days ago I started to get this error:
Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-04-03T05:55:24.847Z] GitHub Copilot could not connect to the server. Error on extension activation: "At this time, Copilot is not available at your location."

I checked my github student developer pack and it is still active, even in january, i received an email that my github copilot was renewed until 2025, I'm from Mexico.

I need help please, thank you.

**Comment by ivan-plotnikov** [comment]
same

**Comment by Akash1134** [comment]
Hi @jose1500s and @ivan-plotnikov ,

Thanks for getting in touch.

Due to current trade controls, your GitHub account has been restricted from making use of Copilot. This is a region-based restriction and applies to all users located in some trade-restricted countries, being flagged for the same earlier, and is separate from other GitHub services. You can continue to use other GitHub services including your Pro plan. You can read more about it here:

[GitHub and Trade Controls](https://docs.github.com/en/site-policy/other-site-policies/github-and-trade-controls)

Sorry if that wasn't what you were hoping to hear. 

If you feel this should not be the case, feel free to contact our Support team and [open a ticket ](https://support.github.com/request?tags=dotcom-footer)with them. Thanks!

**Reply by jose1500s** [reply]
I don't think that's the case, I live in Mexico, I have other friends that it continues to work perfectly for them, I already have a support ticket but they don't respond

**Comment by yamnjjt** [comment]
I had similar issue and installing the extension with the version 1.71.8269 resolved the issue for me.
